src/pages/ErrorPage.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from "react";
+
+class ErrorPage extends React.Component {
+
+    static propTypes = {
+        err: PropTypes.object,
+    }
+
+    render() {
+        const { err } = this.props;
+        return (
+            <div>
+                <h1>Error displaying this page</h1>
+
+                { process.env.NODE_ENV === "development" && err &&
```

Comment:
You mean like `https://www.npmjs.com/package/debug`? I got the impression that's for console debugging rather than this kind of in-page thing.